Backpack/Rating/Classes/BPKRating.swift

```diff
@@ -0,0 +1,263 @@
+        }
+    }
+
+    // MARK: - Subviews
+    private let horizontalStackView: UIStackView = {
+        let horizontalStack = UIStackView()
+        horizontalStack.axis = .horizontal
+        horizontalStack.spacing = BPKSpacingMd
+        horizontalStack.alignment = .lastBaseline
+        return horizontalStack
+    }()
+    private let ratingValueAndScaleStackView: UIStackView = {
+        let horizontalStack = UIStackView()
+        horizontalStack.axis = .horizontal
+        horizontalStack.alignment = .lastBaseline
+        return horizontalStack
+    }()
+    private let titleSubtitleStackView: UIStackView = {
+        let verticalStack = UIStackView()
+        verticalStack.axis = .vertical
+        verticalStack.alignment = .leading
+        verticalStack.distribution = .fill
+        return verticalStack
+    }()
+
+    private let ratingValueLabel: BPKLabel = {
+        let label = BPKLabel(fontStyle: .textLabel1)
+        label.textColor = BPKColor.textPrimaryColor
+        return label
+    }()
+    private let ratingScaleLabel: BPKLabel = {
+        let label = BPKLabel(fontStyle: .textCaption)
+        label.textColor = BPKColor.textSecondaryColor
+        return label
+    }()
+    private let titleLabel: BPKLabel = {
+        let label = BPKLabel(fontStyle: .textLabel1)
+        label.textColor = BPKColor.textPrimaryColor
+        return label
+    }()
+    private let subtitleLabel: BPKLabel = {
+        let label = BPKLabel(fontStyle: .textBodyDefault)
+        label.textColor = BPKColor.textSecondaryColor
+        return label
+    }()
+    private var titleViewBottomConstraint: NSLayoutConstraint?
+
+    // MARK: - Init
+    public init(
+        accessibilityLabel: String,
+        title: String,
+        value: Float,
+        ratingScale: BPKRatingScale = .zeroToFive,
+        size: BPKRatingSize = .default,
+        subtitle: String? = nil,
+        showScale: Bool = true
+    ) {
+        self.value = value
+        self.ratingScale = ratingScale
+        self.size = size
+        self.showScale = showScale
+        self.titleView = titleLabel
+        super.init(frame: .zero)
+
+        self.title = title
+        self.subtitle = subtitle
+        self.accessibilityLabel = accessibilityLabel
+
+        setup()
+        updateLookAndFeel()
+    }
+
+    public init(
+        accessibilityLabel: String,
+        value: Float,
+        ratingScale: BPKRatingScale = .zeroToFive,
+        size: BPKRatingSize = .default,
+        subtitle: String? = nil,
+        showScale: Bool = true,
+        titleView: UIView
+    ) {
+        self.value = value
+        self.ratingScale = ratingScale
+        self.size = size
+        self.showScale = showScale
+        self.titleView = titleView
+
+        super.init(frame: .zero)
+
+        self.subtitle = subtitle
+        self.accessibilityLabel = accessibilityLabel
+
+        setup()
+        updateLookAndFeel()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
```

Comment:
```suggestion
    @available(*, unavailable)
    required init?(coder aDecoder: NSCoder) {
```

Reply:
The related commit is d0ee6ed296b3b29f6054ed34cf8afd7117bfbfcc